### PR TITLE
OHRM5X-1288: fix upgrader UI bugs

### PR DESCRIPTION
**Pull request by Ajiharan007**
this PR contains
OHRM5X-1289
OHRM5X-1290
OHRM5X-1286

### CODE REVIEW

File: installer/client/src/pages/UpgraderCompleteScreen.vue [hunk 1]
```diff
@@ -12,9 +12,9 @@
     <ul class="orangehrm-upgrader-complete">
       <li>
         <oxd-text tag="span" class="orangehrm-installer-page-content">
-          If note were displayed under <b>Version Info,</b> make sure you
-          understand them throughly, You can find the notes at
-          <b>Upgrading/log/notes.log</b> in your OrangeHRM root folder.
+          If notes were displayed under <b>Version Info,</b> make sure you
+          understand them thoroughly, You can find the notes at
+          <b>upgrader/log/notes.log</b> in your OrangeHRM root folder.
```
Comment (RajithaKumara) [inline installer/client/src/pages/UpgraderCompleteScreen.vue:17]:
```suggestion
          If notes were displayed under <b>Version Info,</b> make sure you
          understand them thoroughly, You can find the notes at
          <b>src/log/installer.log</b> in your OrangeHRM root folder.
```

File: installer/client/src/pages/WelcomeScreen.vue [hunk 2]
```diff
@@ -108,6 +108,10 @@ export default {
   },
   methods: {
     toggleModal() {
+      if (this.selected === 'install') {
+        navigate('/../installerUI.php');
+        return;
+      }
```
Comment (RajithaKumara) [inline installer/client/src/pages/WelcomeScreen.vue:114]: not sure requirement. cz `installerUI.php` legacy installer already removed

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: installer/client/src/pages/WelcomeScreen.vue (hunk 2 above)
Comment (Ajiharan007) [reply installer/client/src/pages/WelcomeScreen.vue:114]: then how to redirect to  the installation page. when we click the fresh installation

File: installer/client/src/pages/WelcomeScreen.vue [hunk 3]
```diff
@@ -108,14 +108,16 @@ export default {
   },
   methods: {
     toggleModal() {
+      if (this.selected === 'install') {
+        navigate('/installer/database-config');
```
Comment (Super-Chama) [inline installer/client/src/pages/WelcomeScreen.vue:112]: why this? we do this in closeModal function right? which also check if user has accepted it

File: src/client/tsconfig.json [hunk 4]
```diff
@@ -20,7 +20,6 @@
     "lib": ["esnext", "dom", "dom.iterable", "scripthost"]
   },
   "include": [
-    "src/**/*.ts",
```
Comment (Super-Chama) [inline src/client/tsconfig.json:23]: remove this change

File: installer/client/src/pages/WelcomeScreen.vue (hunk 3 above)
Comment (Ajiharan007) [reply installer/client/src/pages/WelcomeScreen.vue:112]: we don't need to show warning pop up modal when we have a fresh installation

**Review by Super-Chama — APPROVED** [review]

File: installer/Util/SystemConfig.php [hunk 5]
```diff
@@ -383,7 +383,7 @@ public function isMaximumSessionIdle(): array
         $timeSpan = "($gcMaxLifeTimeMinutes minutes and $gcMaxLifeTimeSeconds seconds)";
         if ($gcMaxLifeTimeMinutes > 15) {
             return [
-                'message' => Messages::MAXIMUM_SESSION_IDLE_OK_MESSAGE . $timeSpan,
+                'message' => self::getExtensionsOkMessage(),
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:386]:
```suggestion
                'message' => $this->getExtensionsOkMessage(),
```

Comment (SandeepaOHRM) [reply installer/Util/SystemConfig.php:386]: refer all the ** marked places

File: installer/Util/SystemConfig.php [hunk 6]
```diff
@@ -415,7 +415,7 @@ public function isRegisterGlobalsOff(): array
             ];
         } else {
             return [
-                'message' => Messages::REGISTER_GLOBALS_OFF_OK_MESSAGE,
+                'message' => self::getExtensionsOkMessage(),
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:418]: **

File: installer/Util/SystemConfig.php [hunk 7]
```diff
@@ -430,35 +430,17 @@ public function getAllocatedMemoryStatus(): array
         $hardLimit = 9;
         $softLimit = 16;
         $maxMemory = null;
-        $message = '';
+        $message = self::getExtensionsOkMessage();
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:433]: **

File: installer/Util/SystemConfig.php [hunk 8]
```diff
@@ -471,7 +453,7 @@ public function isCurlEnabled(): array
     {
         if (extension_loaded('curl')) {
             return [
-                'message' => Messages::CURL_STATUS_OK_MESSAGE,
+                'message' => self::getExtensionsOkMessage(),
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:456]: **

File: installer/Util/SystemConfig.php [hunk 9]
```diff
@@ -491,7 +473,7 @@ public function isSimpleXMLEnabled(): array
     {
         if (extension_loaded('SimpleXML') && extension_loaded('libxml') && extension_loaded('xml')) {
             return [
-                'message' => Messages::SIMPLE_XML_STATUS_OK_MESSAGE,
+                'message' => self::getExtensionsOkMessage(),
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:476]: **

File: installer/Util/SystemConfig.php [hunk 10]
```diff
@@ -511,7 +493,7 @@ public function isZipExtensionEnabled(): array
     {
         if (extension_loaded('zip')) {
             return [
-                'message' => Messages::ZIP_STATUS_OK_MESSAGE,
+                'message' => self::getExtensionsOkMessage(),
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:496]: **

File: installer/Util/SystemConfig.php [hunk 11]
```diff
@@ -771,4 +753,12 @@ public function getSystemDetails(): array
             ]
         ];
     }
+
+    /**
+     * @return string
+     */
+    public function getExtensionsOkMessage(): string
```
Comment (SandeepaOHRM) [inline installer/Util/SystemConfig.php:760]:
```suggestion
    private function getExtensionsOkMessage(): string
```

**Review by RajithaKumara — APPROVED** [review]

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**